workflow/Snakefile

```diff
@@ -5,6 +5,7 @@ import yaml
 # open configuration file
 configfile: "workflow/config.yaml"
 configpath = workflow.configfiles[0]
+# TODO How does this work? Where does the `config` variable come from?
```

Comment:
It comes from the `workflow/config.yaml` file. 
snakemake reads the `configfile`  as a dict.

Reply:
Thanks - so snakemake automatically reads whatever file is at the location given by `configfile` and then makes that available as a dict as the `config` variable?

Reply:
exactly